src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/BuildUpToDateCheck.cs

```diff
@@ -168,11 +182,11 @@ private void OnOutputGroupChanged(IImmutableDictionary<string, IOutputGroup> e)
             }
         }
 
-        private void OnChanged(IProjectVersionedValue<Tuple<IProjectSubscriptionUpdate, IProjectImportTreeSnapshot, IProjectSubscriptionUpdate, IImmutableDictionary<string, IOutputGroup>>> e)
+        private void OnChanged(IProjectVersionedValue<Tuple<IProjectSubscriptionUpdate, IProjectImportTreeSnapshot, IProjectSubscriptionUpdate, IImmutableDictionary<string, IOutputGroup>, IProjectItemSchema>> e)
```

Comment:
>Tuple [](start = 54, length = 5)

Is there a reason this can't be a C#7 tuple? `IProjectVersionedValue<(IProjectSubscriptionUpdate subscriptionUpdate, IProjectImportTreeSnapshot importTreeSnapshot, IProjectSubscriptionUpdate otherSubscriptionUpdate)>`